package.json

```diff
@@ -4,7 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prettier-fix": "prettier --write '**/*.{js,ts,jsx,tsx,css,json,md,pug,html}' --plugin='@prettier/plugin-pug'"
+  },
+  "lint-staged": {
+    "**/*.{js,ts,jsx,tsx}": [
+      "prettier --write"
+    ],
+    "**/*.css": [
+      "prettier --write"
+    ],
+    "**/*.{json,md,html,pug}": [
+      "prettier --write --plugin=@prettier/plugin-pug"
```

Comment:
You still don't have `.prettierrc` in the project.
Create it with the desired formatting rules and place the plugin definition into.